src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -746,15 +752,19 @@ private List<PortViewModel> CreateProxyInPorts(IEnumerable<PortModel> groupPortM
                 .ToList();
 
             var newPortViewModels = new List<PortViewModel>();
-            for (int i = 0; i < groupPortModels.Count(); i++)
+            double verticalPosition = 0;
+            foreach (var group in groupPortModels)
             {
-                var model = groupPortModels.ElementAt(i);
-                newPortViewModels.Add(originalPortViewModels[i].CreateProxyPortViewModel(model));
-
-                // calculate new position for the proxy inports.
-                model.Center = CalculatePortPosition(model, i);
+                var originalPort = originalPortViewModels.FirstOrDefault(x => x.PortModel.GUID == group.GUID);
```

Comment:
I'm, confused by then, but not super familiar with this code, when would the entire guid of a group equal the guid of a port model?

Reply:
It's the group's port. I will change the variable to groupPort to avoid confusion. Thanks.